### ISSUE TITLE
Ignoring hashes

### ISSUE THREAD

**Issue by iamvdo** [postcss/postcss-url#23]
In `processDecl`, you ignore url with hashes. What is the reason ?

I have a problem with `@font-face` urls that contains hashes. For example, from font-awesome.css:

```css
@font-face {
  font-family: 'FontAwesome';
  src: url('../fonts/fontawesome-webfont.eot?v=4.3.0');
  src: url('../fonts/fontawesome-webfont.eot?#iefix&v=4.3.0') format('embedded-opentype'), url('../fonts/fontawesome-webfont.woff2?v=4.3.0') format('woff2'), url('../fonts/fontawesome-webfont.woff?v=4.3.0') format('woff'), url('../fonts/fontawesome-webfont.ttf?v=4.3.0') format('truetype'), url('../fonts/fontawesome-webfont.svg?v=4.3.0#fontawesomeregular') format('svg');
  font-weight: normal;
  font-style: normal;
}
```

**Comment by MoOx** [comment]
It's written in the CHANGELOG. It was to ignore svg sprites stack.
https://github.com/postcss/postcss-url/commit/c3a9abcbed33ede323e7dcd6708b8fdb6168462f
But maybe it only make sense for svg. We should add more test about that.
poke @bfred-it

**Comment by iamvdo** [comment]
Oh sorry.

I think that urls with hashes should not be ignored, even if they are SVG fragments. Instead, it will be better to store hashes and append it to the new rebased url at the end. 

Why do you think ?

**Comment by MoOx** [comment]
True. Does it make sense to you @bfred-it ?

**Comment by bfred-it** [comment]
Hmm I only use the plugin in inline mode and if there's a hash it can't be inlined. I guess this restriction should only apply if the mode is `inline`, yeah.

**Comment by MoOx** [comment]
Fix released in 2.0.1
